.pa11yci.js

```diff
@@ -0,0 +1,20 @@
+const relativeUrls = require('./pa11y-ci-urls');
```

Comment:
The alternative implementation would be to have an upstream text file with a list of urls on each line, then use node fs.open to read and process.
By having the file generated with an (e.g.) `.txt` extension rather than a `.js` extension, it's easier for someone to recognise that it doesn't contain logic, only data.

Reply:
I've had a go, but it's a pain in the bum, because there's no easy way in Node.js to read a file line by line synchronously. (Asynchronously is easy, but no use, since `module.exports` can't be done asynchronously.) The options are:

* add a dependency ([readline-sync](https://www.npmjs.com/package/readline-sync))
* implement our own line-reading logic (perhaps including worrying about the different variant line breaks you might get on different systems)

Against those two options, I think I prefer keeping it as it is